rpm-ostree-bisect

```diff
@@ -166,17 +199,22 @@ def initialize_commits_info(repo, bad, good):
 
     return info
 
+
 """
     Grab all commit history from the remote (just
     the metadata).
 """
 def pull_commit_history(deployment, repo):
 
     # Get repo, remote and refspec from the booted deployment
+    # The refspec in the metadata is either `refspec` if there
+    # are no layered packages (i.e. the deployed commit is a base
+    # commit) or `baserefspec` if there are.
     origin = deployment.get_origin()
-    refspec = origin.get_string('origin', 'refspec')
-    # with layered packages it has baserefspec
-    #refspec = origin.get_string('origin', 'baserefspec')
+    if is_base_commit(repo, deployment.get_csum()):
+        refspec = origin.get_string('origin', 'refspec')
+    else:
+        refspec = origin.get_string('origin', 'baserefspec')
```

Comment:
Once packages are layered, even if they're unlayered, it'll stay `baserefspec`. So here instead of basing it off of `is_base_commit`, we should just always check for `refspec` then fall back to `baserefspec`.

Reply:
ok will check for that